api/types/types.go

```diff
@@ -219,18 +222,29 @@ type InfoBase struct {
 	ContainerdCommit   Commit
 	RuncCommit         Commit
 	InitCommit         Commit
+	SecurityOptions    []string
 }
 
-// SecurityOpt holds key/value pair about a security option
 type SecurityOpt struct {
 	Key, Value string
 }
 
-// Info contains response of Remote API:
-// GET "/info"
-type Info struct {
-	*InfoBase
-	SecurityOptions []SecurityOpt
+func DecodeSecurityOptions(opts []string) ([]SecurityOpt, error) {
+	so := []SecurityOpt{}
+	for _, opt := range opts {
+		split := strings.Split(opt, ",")
```

Comment:
To support the output from a 1.12 docker daemon, we could check if the string does not contain a `"="` character in which case we could probably just do something like this:

```go
so := []SecurityOpt{}
for _, opt := range opts {
    if !strings.Contains(opt, "=") {
        so = append(so, SecurityOpt{Key: "name", Value: opt})
        continue
    }
    // Continue with the regular logic.
    ...
}
```